src/renderer.zig

```diff
@@ -917,8 +917,23 @@ pub const MeshSelection = struct { // MARK: MeshSelection
 		while(total_tMax < closestDistance) {
 			const block = mesh_storage.getBlock(voxelPos[0], voxelPos[1], voxelPos[2]) orelse break;
 			if(block.typ != 0) blk: {
-				for(block.blockTags()) |tag| {
-					if(tag == .fluid or tag == .air) break :blk; // TODO: Buckets could select fluids
+				var skipTagCheck = false;
```

Comment:
Please use a more descriptive name. "tag check" is not really giving me a clue on what this is actually skipping.

Why not just call it `fluidPlaceable`, like the property it reflects?